lib/livebook_web/live/hub/new_live.ex

```diff
@@ -137,30 +137,65 @@ defmodule LivebookWeb.Hub.NewLive do
               label="Livebook Teams Key"
             />
 
-            <div :if={@requested_code} class="grid grid-cols-1 gap-3">
-              <span><%= @request_code_info %></span>
-
-              <.link navigate={@verification_uri} target="_blank" class="font-bold text-blue-500">
-                <%= @verification_uri %>
-              </.link>
-
-              <span><%= @org.user_code %></span>
-            </div>
-
             <button
               :if={!@requested_code}
               class="button-base button-blue self-start"
-              phx-disable-with="Creating..."
+              phx-disable-with="Loading..."
             >
               <%= @button_label %>
             </button>
+            <div class="invisible"></div>
```

Comment:
Do we need this `div`?

Reply:
Yes, it’s taking the left space so the code can be centered and the clipboard icon is on the right.

Reply:
@paulo-valim that's not the invisible icon though (it's in the `copyclip` component), right?

Reply:
No, it's just an empty space, so the flex justify-around works out with 3 elements.
The invisible class is to say what it's.

Reply:
Right, but the one for centering clipboard icon is [here](https://github.com/livebook-dev/livebook/pull/1935/files#diff-bd44ad419ea963b24a512dbdd24c4efc24c2a82a0f2e2df1d0bdf6459fff9ad7R179-R181)?

Reply:
Yes, it's this one:

`  <div class="icon-button invisible">
        <.remix_icon icon="clipboard-line " class="text-lg" />
      </div>`

Reply:
So what does the other invisible div do? :D